sn_cli/src/subcommands/files/mod.rs

```diff
@@ -251,40 +265,7 @@ async fn upload_files(
     info!("Made payment of {total_cost} for {unpaid_chunks_to_upload_len} chunks");
     info!("New wallet balance: {final_balance}");
 
-    // If we are not verifying, we can skip this
-    if verify_store && !verify_as_we_upload {
-        println!("**************************************");
-        println!("*            Verification            *");
-        println!("**************************************");
-
-        let mut attempts = 0;
-
-        // There will be MAX_CHUNK_PUT_RETRIES (currently 10) re-puts
-        // within each `verify_and_repay_if_needed` run.
-        // To avoid infinite looping here due to other unrecoverable error,
-        // the `while` loop need to be terminated after certain attempts.
-        // Meanwhile, with real network, there will be certain chunks need to be re-put,
-        // especially with large files/folder.
-        // Note those succeeded files will still get recorded within `uploaded_files`,
-        // after complete this while loop of verification.
-        while !chunk_manager.is_paid_chunks_empty() && attempts <= MAX_VERIFICATION_RETRIES {
-            attempts += 1;
-            println!("The current overall verify and repay iteration is {attempts}");
-            verify_and_repay_if_needed(
-                file_api.clone(),
-                &mut chunk_manager,
-                batch_size,
-                show_holders,
-            )
-            .await?;
-        }
-    } else {
-        // for safe measure, mark all as paid too
-        chunk_manager.mark_paid_all();
-        chunk_manager.mark_verified_all();
-    }
```

Comment:
If we didn't have verification on, then the swallowed errors would just be considered as success! With no way for the users to even manually retry as the chunks are marked as OK even though they might not!